src/main/java/com/conveyal/datatools/editor/datastore/SnapshotTx.java

```diff
@@ -66,7 +67,11 @@ public void make (FeedTx master) {
         try {
             targetTx.getAll();
         } catch (RuntimeException e) {
-            LOG.error("Target FeedTX for feed restore may be corrupted.  Consider wiping feed database editor/$FEED_ID/master.db*", e);
+            LOG.error("Target FeedTX for feed restore may be corrupted.  Wiping feed database (it will be restored with a snapshot anyways).", e);
```

Comment:
The context here (and in the pull request comments) is not clear to me. What is a "feed database"? What is the difference between "wiping" it and what it did in the past? I think this means that we're deleting the files instead of replacing all the tables within the file. Maybe we should just do that every time.

Reply:
I'm not sure we should delete the db files every time.  The db still stores the snapshot version, which we would ideally preserve, rather than iteratively re-incrementing it to get to the latest available version.